setup.py

```diff
@@ -96,14 +96,14 @@ def package_files(directory):
     author='Chris Higgs, Stuart Hodgson',
     maintainer='cocotb contributors',
     maintainer_email='cocotb@lists.librecores.org',
-    install_requires=[],
+    install_requires=['find_libpython'],
```

Comment:
We could copy the script locally to prevent a dependency. Or make this optional and if you don't have it, all you get is `libpython.so` so it resolves as it otherwise would?